modules/migrating-to-multi-arch-cli.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * updating/migrating-to-multi-payload.adoc
+
+:_content-type: PROCEDURE
+[id="migrating-to-multi-arch-cli_{context}"]
+= Migrating to a cluster with multi-architecture compute machines using the CLI
+
+.Prerequisites
+
+* You must be logged in as a user with `cluster-admin` privileges on all clusters.
+* Your {product-title} version must be up to date to at least version 4.13.0. 
++
+For more information on how to update your cluster version, see _Updating a cluster using the web console_ or _Updating a cluster using the CLI_.
+* You installed the OpenShift CLI (`oc`).
+* Your oc client must be updated to at least verion 4.13.0. 
+* Your {product-title} cluster must be installed on either the AWS or Azure platform. 
++
+For more information on selecting a supported platform for your cluster installation, see _Selecting a cluster installation type_.
+
+
+.Procedure
+. Verify `RetrievedUpdates=True` in Cluster Version Operator (CVO) conditions by running the following command:
++
+[source,terminal]
+----
+$ oc adm upgrade
+----
++ 
+For more information about cluster version condition types, see _Understanding cluster version condition types_.
+
+. Set the channel to `stable-<4.y>` or `fast-<4.y>` with the following command: 
++
+[source,terminal]
+----
+$ oc adm upgrade channel <channel>
```

Comment:
The procedure to me is like, we verify if the condition `RetrievedUpdates=True` , if not, we switch channel till we see condition `RetrievedUpdates=True`. Otherwise, it's not mandatory to change the channel.

Reply:
Ah :) thank you!

Reply:
I haven't gone through all of the other discussion on this pull request,  so I may be missing context.  But while `NoChannel`  is one source of `RetrievedUpdates=False`, and `VersionNotFound` for a stock install's default `stable-4.y` channel in the days after its errata ships before it enters `stable-4.y`.  But selecting a channel is an important decision [with its own long page of context][1], and while I'm not clear on what to do about all the ways `RetrievedUpdates` can go wrong, I'm pretty sure we don't want a oneliner here suggesting folks change channels.

[1]: https://docs.openshift.com/container-platform/4.12/updating/understanding-upgrade-channels-release.html